src/main/java/org/spectrumauctions/sats/core/model/mrvm/MRVMWorld.java

```diff
@@ -67,7 +67,7 @@ public int getNumberOfGoods() {
         for (MRVMBand band : bands) {
             licenses.addAll(band.getLicenses());
         }
-        return licenses;
+        return ImmutableSet.copyOf(licenses);
```

Comment:
Same as above. To keep the review simple, I will not mark the other equivalent lines in other classes.

Reply:
Makes sense, I took care of the other equivalent cases